.ci.yaml

```diff
@@ -81,6 +81,8 @@ targets:
     timeout: 90
     properties:
       add_recipes_cq: "true"
+    runIf:
+      - packages/fuchsia_ctl/
```

Comment:
You probably also want to add https://github.com/flutter/packages/tree/main/.ci and .ci.yaml as triggers as well

Reply:
Good call, did those and `script/`.